apps/condo/domains/user/components/containers/PosterLayoutVertical.tsx

```diff
@@ -0,0 +1,67 @@
+import styled from '@emotion/styled'
```

Comment:
Unused import?

Reply:
It was an old unused component, that was firstly created and left orphaned